### DISCUSSION TITLE
Some of the font awesome Icons are not visible

### DISCUSSION THREAD

**Discussion by DrastiGohel** [serenity-is/Serenity#6586] [Help]
Hello All,

After we migrated to new serenity version (6.2.2), some of font awesome icons are not visible

For example in old version this icon was used for edit implace:
![image](https://user-images.githubusercontent.com/98172041/203097979-d8bfa42c-dd2e-47c8-8b49-a9a268d3d843.png)

In migrated code same icon is used but it is not visible:
![image](https://user-images.githubusercontent.com/98172041/203099057-20d35025-87f0-4d1b-bc26-299b16816639.png)

Other font awesome icons on the same page are visible

**Comment by VictorTomaili** [comment]
pro-theme is StartSharp feature, you may miss some step while upgrading. Check included font styles. And please use StartSharp repository for StartSharp related support.

**Reply by DrastiGohel** [reply]
Hi Victor,

I will look for it. And I will use StartSharp from next time

Thanks
